Cargo.toml

```diff
@@ -37,8 +38,8 @@ mpc-tls = { path = "crates/tls" }
 node-types = { path = "crates/node-types" }
 tee_authority = { path = "crates/tee_authority" }
 test_utils = { path = "crates/test_utils" }
+utilities = { path = "crates/utilities" }
 
-actix = "0.13.0"
```

Comment:
Oh I've been longing for this day. Thank you Actix for your faithful service :saluting_face: you may retire now.